draft-esdih-ccamp-layer0-types-ext-00.txt

```diff
@@ -0,0 +1,1344 @@
+
+
+
+
+CCAMP                                                     D. Beller, Ed.
+Internet-Draft                                           S. Belotti, Ed.
+Updates: draft-ietf-ccamp-layer0-types (if                         Nokia
+         approved)                                              H. Zheng
+Intended status: Standards Track                                 I. Busi
+Expires: 26 August 2021                                           Huawei
+                                                            E. Le Rouzic
+                                                                  Orange
+                                                        22 February 2021
+
+
+            A YANG Data Model for Layer 0 Types - Revision 2
+                 draft-esdih-ccamp-layer0-types-ext-00
+
+Abstract
+
+   This document defines a collection of common data types and groupings
+   in the YANG data modeling language, which are used in several YANG
+   modules for wavelength Division multiplexing (WDM) transport
+   networks.  The YANG module ietf-layer0-types-ext updates ietf-
+   layer0-types [I-D.ietf-ccamp-layer0-types], which has been reduced in
+   scope prior to publication to only cover spectrum management related
+   aspects required for ietf-wson-topology [I-D.ietf-ccamp-wson-yang].
```

Comment:
> I do not think we need to reference the history and the WSON topology model here.
> 
> Moreover, idnits complains with references in the abstract.

Maybe the reference of wson topology is not correct, but it is important to underline what is the reason and the context of this extension. The fact that layer0-type has been reduced  "to only cover spectrum management related
   aspects" it is important and maybe we need to extend a bit that. If you have some suggestion for that with text ,  it could be added in next updated.